exonum/src/storage/proof_map_index/key.rs

```diff
@@ -32,30 +32,118 @@ pub const PROOF_PATH_LEN_POS: usize = KEY_SIZE + 1;
 /// A trait that defines a subset of storage key types which are suitable for use with
 /// `ProofMapIndex`.
 ///
-/// The size of the keys must be exactly 32 bytes and the keys must have a uniform distribution.
-pub trait ProofMapKey: StorageKey {}
+/// The size of the keys must be exactly [`PROOF_MAP_KEY_SIZE`] bytes and the keys must have
+/// a uniform distribution.
+///
+/// [`PROOF_MAP_KEY_SIZE`]: constant.PROOF_MAP_KEY_SIZE.html
```

Comment:
Here and everywhere else. Try to open `PROOF_MAP_KEY_SIZE` in docs - it turns out it's impossible to learn actual value of this constant from docs.